### DISCUSSION TITLE
About L-BFGS

### DISCUSSION THREAD

**Discussion by Dingiso-cyber** [lululxvi/deepxde#1835] [Q&A]
Hi, Dr. LU, thanks for your contribution to the ML society. Deepxde is great! 
Question: when I use the Callbacks PDEPointResampler, I find it works well with the Adam method, like my training set is changed every period, but it doesn't work when I compile the model with L-BFGS method, the training set is not change at all. Is it possible to fix this? PS: I use the pytorch as the backend.

**Comment by lululxvi** [comment]
PDEPointResampler doesn't work with L-BFGS.
